### PR TITLE
updated login.spec.js and uses best practice of cypress 

### PR DESCRIPTION
**Pull request by manzil-infinity180**
Explained each and every thing - #794

https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/119070053/ca1872b7-da73-4d70-83dc-cc402ab554d2

Screenshot 2024-03-14 at 1 05 08 AM

Screenshot 2024-03-14 at 1 20 07 AM

### CODE REVIEW

File: frontend/cypress/e2e/login.cy.js [hunk 1]
```diff
@@ -30,6 +26,11 @@ describe('Failing or Succeeding to Login', function () {
                     cy.get('div[role=\'status\']').should('be.visible');
                 }
             });
+            cy.wait(2000);
+            cy.url().should('include','/');
+            cy.get('[data-test="panelSwitchIcon"').should('be.visible');
+            cy.get('[data-test="openMenu"').should('be.visible');
+            cy.get('[data-test="logo"').should('be.visible').should('have.id','header-logo');
         });
```
Comment (sherrif10) [inline frontend/cypress/e2e/login.cy.js:34]: Add another step for a successfull login atleast to ensure login part is done. Then this should be good to be merged.

Comment (manzil-infinity180) [reply frontend/cypress/e2e/login.cy.js:34]: @sherrif10 login has already done i just refractor and will add more to it

**Comment by tendomart** [comment]
@manzil-infinity180  some conflicts here

**Comment by manzil-infinity180** [comment]
> @manzil-infinity180 some conflicts here

how i can resolve it ?

**Comment by tendomart** [comment]
You can hit the Resolve conflicts button, it will open an editor so you can delete some unwanted lines.

**Comment by mozzy11** [comment]
Closing this because theres no update for some time. 
@manzil-infinity180 feel free to re-open it and adrees the comments

**Pull request closed without merging**